server/src/main/java/io/spine/server/aggregate/AggregateRepository.java

```diff
@@ -352,7 +352,7 @@ protected final void onRoutingFailed(SignalEnvelope<?, ?, ?> envelope, Throwable
 
     @Override
     public Set<EventClass> domesticEvents() {
-        return aggregateClass().domesticEvents();
+        return aggregateClass().events();
```

Comment:
Why does the `domesticEvents()` method of `AggregateRepository` delegates to the (all) `events()` method of aggregate class? If this is not a typo, could you please clarify vocally or in doc?

Reply:
Renamed to `events()`.